[2025-04-18T20:04:41.125+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_apod_postgres.create_table manual__2025-04-18T20:04:40.277027+00:00 [queued]>
[2025-04-18T20:04:41.130+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_apod_postgres.create_table manual__2025-04-18T20:04:40.277027+00:00 [queued]>
[2025-04-18T20:04:41.131+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-18T20:04:41.138+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2025-04-18 20:04:40.277027+00:00
[2025-04-18T20:04:41.141+0000] {standard_task_runner.py:60} INFO - Started process 374 to run task
[2025-04-18T20:04:41.143+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'nasa_apod_***', 'create_table', 'manual__2025-04-18T20:04:40.277027+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpk0wfar7_']
[2025-04-18T20:04:41.144+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask create_table
[2025-04-18T20:04:41.153+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T20:04:41.172+0000] {task_command.py:423} INFO - Running <TaskInstance: nasa_apod_postgres.create_table manual__2025-04-18T20:04:40.277027+00:00 [running]> on host 4d4fb407ffc6
[2025-04-18T20:04:41.215+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='nasa_apod_***' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T20:04:40.277027+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T20:04:40.277027+00:00'
[2025-04-18T20:04:41.220+0000] {base.py:83} INFO - Using connection ID 'my_***_connection' for task execution.
[2025-04-18T20:04:41.222+0000] {sql.py:450} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS apod_data (
            id SERIAL PRIMARY KEY,
            title VARCHAR(255),
            explanation TEXT,
            url TEXT,
            date DATE,
            media_type VARCHAR(50)
        );


        , parameters: None
[2025-04-18T20:04:41.222+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-18T20:04:41.227+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=nasa_apod_***, task_id=create_table, execution_date=20250418T200440, start_date=20250418T200441, end_date=20250418T200441
[2025-04-18T20:04:41.275+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-18T20:04:41.294+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
